openstack/02b-MAChigh/schedule.c

```diff
@@ -461,6 +464,45 @@ bool schedule_isSlotOffsetAvailable(uint16_t slotOffset){
    return TRUE;
 }
 
+uint8_t schedule_getUsageStatus(scheduleEntry_t* entry){

```

Comment:
add a comment. I don't understadn what a "usage status" is. The name is not clear.